test/compilable/extra-files/ddoc17697.html

```diff
@@ -1,484 +1,506 @@
-
```

Comment:
Oh I understand, thanks. Yeah that's a big and surprising change, will try to keep endlines as they were.